Drop after FreeBSD 11 EOL. OpenSSL 1.0 still works fine.

-- Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the system variable OPENSSL_ROOT_DIR: Found unsuitable version "1.0.2u", but required is at least "1.1" (found /usr/lib/libcrypto.so, )
CMake Error at externals/CMakeLists.txt:83 (add_subdirectory):

In file included from src/core/hle/service/bcat/backend/boxcat.cpp:6:
externals/./httplib/httplib.h:214:2: error: Sorry, OpenSSL versions prior to 1.1.1 are not supported
#error Sorry, OpenSSL versions prior to 1.1.1 are not supported
 ^

--- externals/CMakeLists.txt.orig	2020-11-18 04:02:27 UTC
+++ externals/CMakeLists.txt
@@ -73,7 +73,7 @@ if (NOT LIBZIP_FOUND)
 endif()
 
 if (ENABLE_WEB_SERVICE)
-    find_package(OpenSSL 1.1)
+    find_package(OpenSSL)
     if (OPENSSL_FOUND)
         set(OPENSSL_LIBRARIES OpenSSL::SSL OpenSSL::Crypto)
     else()
--- externals/cpp-httplib/httplib.h.orig	2021-06-18 12:45:50 UTC
+++ externals/cpp-httplib/httplib.h
@@ -219,10 +219,6 @@ using socket_t = int;
 #include <iostream>
 #include <sstream>
 
-#if OPENSSL_VERSION_NUMBER < 0x1010100fL
-#error Sorry, OpenSSL versions prior to 1.1.1 are not supported
-#endif
-
 #if OPENSSL_VERSION_NUMBER < 0x10100000L
 #include <openssl/crypto.h>
 inline const unsigned char *ASN1_STRING_get0_data(const ASN1_STRING *asn1) {
@@ -6729,7 +6725,11 @@ static SSLInit sslinit_;
 inline SSLServer::SSLServer(const char *cert_path, const char *private_key_path,
                             const char *client_ca_cert_file_path,
                             const char *client_ca_cert_dir_path) {
+#if OPENSSL_VERSION_NUMBER < 0x10100001L
+  ctx_ = SSL_CTX_new(SSLv23_server_method());
+#else
   ctx_ = SSL_CTX_new(TLS_method());
+#endif
 
   if (ctx_) {
     SSL_CTX_set_options(ctx_,
